db/upgrade-X.X-X.Y.sql

```diff
@@ -67,6 +67,9 @@ ALTER TABLE `pki_certs`
 \! echo "Adding default timestamp to RADIUS audit logs";
 ALTER TABLE radius_audit_log MODIFY created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP;
 
+\! echo "Make psk unique";
+ALTER TABLE person ADD CONSTRAINT UNIQUE person_psk (`psk`);
```

Comment:
This statement will fail if there's more than one row with the same psk. Before this statement we should first empty it to avoid duplicates: `UPDATE USERS SET psk=NULL WHERE 1;`

Reply:
Yes but i can't empty it first since the psk are actively used